api/apps/geoprocessing/src/legacy-project-import/legacy-piece-importers/features.legacy-piece-importer.ts

```diff
@@ -355,4 +382,39 @@ export class FeaturesLegacyProjectPieceImporter
         : undefined,
     };
   }
+
+  private async updateAmountMinMaxForAPIFeatures(
+    apiEntityManager: EntityManager,
+    featureIds: string[],
+  ): Promise<void> {
+    this.logger.log(`Saving min and max amounts for new features...`);
+
+    const minAndMaxAmountsForFeatures = await this.geoprocessingEntityManager
+      .createQueryBuilder()
+      .select('feature_id', 'id')
+      .addSelect('MIN(amount)', 'amountMin')
+      .addSelect('MAX(amount)', 'amountMax')
+      .from('feature_amounts_per_planning_unit', 'fappu')
+      .where('fappu.feature_id IN (:...featureIds)', { featureIds })
+      .groupBy('fappu.feature_id')
+      .getRawMany();
+
+    const minMaxSqlValueStringForFeatures = minAndMaxAmountsForFeatures
+      .map(
+        (feature) =>
+          `(uuid('${feature.id}'), ${feature.amountMin}, ${feature.amountMax})`,
+      )
+      .join(', ');
+
+    const query = `
+        update features set
+           amount_min = minmax.min,
+           amount_max = minmax.max
+        from (
+        values
+            ${minMaxSqlValueStringForFeatures}
+        ) as minmax(feature_id, min, max)
+        where features.id = minmax.feature_id;`;
+    await apiEntityManager.query(query);
+  }
```

Comment:
ok to keep this in this PR - in a later one, I suggest to use a single instance of this code, after adding a parameter to allow to pass in an apiEntityManager and removing the reliance on any entity managers injected in the parent class, so that we can remove duplicated code... though if moving this to an existing lib (for example `libs/geofeature-calculations`) turns out to be even just slightly more than banal, duplication be it and let's move on with life.